api_tests/lib/ledger_runner.ts

```diff
@@ -0,0 +1,103 @@
+import * as bitcoin from "./bitcoin";
+import * as toml from "toml";
+import * as fs from "fs";
+import { sleep } from "./util";
+import { execSync, spawn } from "child_process";
+
+const project_root: string = execSync("git rev-parse --show-toplevel", {
+    encoding: "utf8",
+}).trim();
+const docker_cwd = project_root + "/api_tests/regtest";
```

Comment:
It looks like you forgot to delete these lines :)

Reply:
😇